runtimes/rust/src/lib.rs

```diff
@@ -262,31 +262,30 @@ pub use self::{
 pub mod api;
 pub mod error;
 
-use exonum_merkledb::Snapshot;
-use futures::{future, sync::mpsc, Future, IntoFuture, Sink};
-
-use std::collections::{BTreeMap, HashMap, HashSet};
-
-use crate::{
+use exonum::{
     api::{manager::UpdateEndpoints, ApiBuilder},
     blockchain::{Blockchain, Schema as CoreSchema},
     helpers::Height,
     runtime::{
-        dispatcher::{self, Mailbox},
-        error::{catch_panic, ExecutionError},
-        ArtifactId, BlockchainData, CallInfo, ExecutionContext, InstanceDescriptor, InstanceId,
-        InstanceSpec, InstanceStatus, Runtime, RuntimeIdentifier, WellKnownRuntime,
+        catch_panic, ArtifactId, BlockchainData, CallInfo, DispatcherError, ExecutionContext,
+        ExecutionError, InstanceDescriptor, InstanceId, InstanceSpec, InstanceStatus, Mailbox,
+        Runtime, RuntimeIdentifier, WellKnownRuntime,
     },
 };
+use exonum_merkledb::Snapshot;
+use futures::{future, sync::mpsc, Future, IntoFuture, Sink};
+use log::trace;
+
+use std::collections::{BTreeMap, HashMap, HashSet};
 
 use self::api::ServiceApiBuilder;
 
 mod call_context;
 mod runtime_api;
 mod service;
 mod stubs;
-#[cfg(test)]
-mod tests;
+// #[cfg(test)]
+// mod tests;
```

Comment:
I suggest to add some warning-spawning code here, because it's really easy to forget to uncomment this later, e.g.:

```rust
// TODO: This is placed here so warning will be spawned
// until code below is uncommented.
const FIXME: u8 = 42;
// #[cfg(test)]
// mod tests;
```